internal/types/document.go

```diff
@@ -23,6 +23,11 @@ import (
 	"github.com/FerretDB/FerretDB/internal/util/must"
 )
 
+const (
+	// IdKey is the key for _id field.
+	IdKey = "_id"
```

Comment:
In the file `document_validation.go` ids in consts are called ID (capital letters), so I guess here those should be capital letters too?

```suggestion
	IDKey = "_id"
```